CONTRIBUTING.md

```diff
@@ -350,7 +350,23 @@ by running the test suite in a Docker environment that simulates Swift on Linux.
     cd swift-docc
     swift run docc
     ```
-    
+
+## Updating Build Rules
+
+In order to build DocC as part of the toolchain distribution, a parallel CMake
+based build exists. Note that this is **not** supported for development purposes
+(you cannot execute the test suite with this build).
+
+CMake requires that the full file list is kept up-to-date. When adding or
+removing files in a given module, the `CMakeLists.txt` list must be updated to
+the file list.
+
+You can use the following 1-line script to enumerate the files in the module:
+
+```bash
+python -c "print('\n'.join((f'{chr(34)}{path}{chr(34)}' if ' ' in path else path) for path in sorted(str(path) for path in __import__('pathlib').Path('.').rglob('*.swift'))))"
```

Comment:
I get a syntax error if I try to run this:
```
  File "<string>", line 1
    print('\n'.join((f'{chr(34)}{path}{chr(34)}' if ' ' in path else path) for path in sorted(str(path) for path in __import__('pathlib').Path('.').rglob('*.swift'))))
                                               ^
SyntaxError: invalid syntax
```

Reply:
Can you explain how this python snippet is different from a simpler `find` command like this
```
find . -type f -name "*.swift" | sort
```